docs/usage-tui.md

```diff
@@ -5,107 +5,106 @@ emulator.
+selected), then to the previous wallet. Ctrl-Down will always go to the next wallet. Finally, `h`,
 `j`, `k`, `l` keys will work exactly as Left, Down, Up and Right arrow keys (without Ctrl).
 
-To create a new wallet, select `[ + Add wallet ]` line in the tree, so that a special widget appears
-to the right. There you can either create a completely new wallet by specifying its name and a
+To create a new wallet, select the `[ + Add wallet ]` line in the tree. A special widget will appear
+on the right. There you can either create a completely new wallet by specifying its name and
 passphrase, or restore an existing one. When a new wallet is created, Ariadne will print you its
 mnemonic. **Please save it somewhere safe**.
 
-Note that Ariadne mnemonics have more words than 12-words mnemonics of Daedalus, and that the last
-word of the mnemonic is always fixed (currently it is `ariadne-v0`).  This means you won't be able
+Note that Ariadne mnemonics have more words (13) than mnemonics of Daedalus (12), and that the last
+word of the mnemonic is always fixed (currently it is `ariadne-v0`). This means you won't be able
 to use your Ariadne wallet in Daedalus. However, you can import your Daedalus wallet into Ariadne
-using the 12 words mnemonic.
+using the 12 word mnemonic.
 
-When you select a wallet or an account, you can see its balance and send some transactions with a
-special form.  You must fill at least one receiver's address. To add or remove address lines use
+When you select a wallet or an account, you can see its balance and send transactions using the form provided. You must fill at least one receiver's address. To add or remove address lines, use
 `[ - ]` / `[ + ]` buttons on the right. If the wallet has a passphrase, you need to specify that as
 well. When sending a transaction from a wallet, you can also choose which accounts to use as inputs
 using a list just above the form. Use Enter, Space or mouse click to select accounts.
 
 When an account is selected, you will see the list of its addresses below the send form. If you
-focus an address, you can copy it to the clipboard by pressing Enter, Space or clicking with the
+focus an address, you can copy it to the clipboard by pressing Enter, Space, or clicking with the
 mouse. On macOS this will work out of the box, on Linux you have to have either `xsel` or `xclip`
 installed. To make a new address, press the `[ Generate ]` button.
 
-On the bottom half of the screen you see the REPL widget which consists of command history and an
-input line. The REPL lets you do all sorts of things with your wallet using `Knit` command language.
-In fact, all operations you do with widgets are mapped to some `Knit` commands.
+On the bottom half of the screen, you can see the REPL widget, which consists of command history and an
+input line. The REPL lets you to do all sorts of things with your wallet using the `Knit` command language.
```

Comment:
> (transitive) To allow to, not to prevent (+ infinitive, but **usually without to**).
> After he knocked for hours, I decided to let him come in.

https://en.wiktionary.org/wiki/let#Verb

I'm not sure though.